src/templates/static/legal/metatrader/terms_and_conditions.jsx

```diff
@@ -47,7 +47,7 @@ const TermsAndConditions = () => (
                             <li>{it.L('You have read this Legal Terms and Conditions page in full and understood that you will be buying and selling financial contracts subject to these terms and conditions;')}</li>
                             <li>{it.L('You have read our privacy statement and hereby give us consent to process your personal information as outlined there;')}</li>
                             <li>{it.L('You are over 18 years of age, unless you are an Estonian resident whereby you would have to be over 21;')}</li>
-                            <li>{it.L('You are not resident in a restricted country such as Costa Rica, Hong Kong, Israel, Jersey, Malaysia, Malta, USA or any other restricted country which has been identified by the FATF as having strategic deficiencies;')}</li>
+                            <li>{it.L('You are not resident in a restricted country such as Costa Rica, Hong Kong, Israel, Jersey, Malaysia, Malta, USA, Canada or any other restricted country which has been identified by the FATF as having strategic deficiencies;')}</li>
```

Comment:
The list seems to be alphabetical, shouldn't we follow the same ?

Reply:
yes, i think we should @rakshit-binary